Kudu.Core/Tracing/KuduEventSource.cs

```diff
@@ -60,5 +60,23 @@ public void WebJobEvent(string siteName, string jobName, string Message, string
                 WriteEvent(65513, siteName, jobName, Message, jobType, error);
             }
         }
+
+        [Event(65514, Level = EventLevel.Warning, Message = "Generic event for site {0}", Channel = EventChannel.Operational)]
+        public void GenericEvent(string siteName, string Message, string requestId, string scmType, string siteMode, string buildVersion)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65514, siteName, Message, requestId, scmType, siteMode, buildVersion);
```

Comment:
Yes all to Kudu table.  I have the sample on separate email thread (title Add kudu traces to Geneva (Kusto)).